doc/nrf/libraries/caf/ble_adv.rst

```diff
@@ -98,22 +63,29 @@ This lets you speed up finding the Peripheral by Bluetooth Centrals.
 
 Switching to slower advertising is done to reduce the energy consumption.
 
-Using Swift Pair
-================
-
-You can use the :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR` option to enable the Swift Pair feature.
-The feature simplifies pairing the Bluetooth Peripheral with Windows 10 hosts.
-
-.. note::
-   Make sure to add the Swift Pair data to advertising packets for unbonded device in the configuration file if you enable :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR` option.
-   The Swift Pair data must be added as the last member of ``ad_unbonded`` array.
-
 Power-down
 ==========
 
 When the system goes to the Power-down state, the advertising stops.
 
-If the Swift Pair feature is enabled with :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR`, the device advertises without the Swift Pair data for additional :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR_GRACE_PERIOD` seconds to ensure that the user does not try to connect to the device that is no longer available.
+Grace period
+------------
+
```

Comment:
Hm, maybe keep line 116 about :kconfig:option:`CONFIG_CAF_BLE_ADV_SWIFT_PAIR_GRACE_PERIOD`? There is suddenly no info about what it does...

Reply:
kconfig:option:CONFIG_CAF_BLE_ADV_SWIFT_PAIR_GRACE_PERIOD was removed.
Now we introduce a generic grace period feature described below. Currently Swift Pair is only one of the use-cases for it.

Reply:
Currently we describe behavior here and then inform about the Kconfig option that is needed here.

Reply:
OK, but then we would need at least one short line introducing the concept of the grace period, information that was present in the removed part: "the device advertises without the Swift Pair data for additional ... seconds to ensure that the user does not try to connect to the device that is no longer available."